Web/starter_project_barbies/tailwind.config.js

```diff
@@ -12,8 +12,9 @@ module.exports = {
         primary: '#264FAD',
         accent: '#FF9F43',
         danger: '#b91c1c',
-        'primary-text': '#353535',
-        'secondary-text': '#717171',
+        primarytext: '#353535',
+        secondarytext: '#717171',
+        darkwhite:"#F6F6FC"
```

Comment:
[nit] Can you add space after colon? It's better to make it consistent with the others.

Reply:
I think darkwhite is vagues. It's better to give it a clear and descriptive name.